src/tests/krate.rs

```diff
@@ -986,16 +985,14 @@ fn yank() {
 #[test]
 fn yank_not_owner() {
     let (_b, app, middle) = ::app();
-    let mut req = ::req(app, Method::Delete, "/api/v1/crates/foo_not/1.0.0/yank");
-    ::mock_user(&mut req, ::user("foo"));
-    ::mock_crate(&mut req, ::krate("foo_not"));
-    ::mock_user(&mut req, ::user("bar"));
+    let mut req = ::request_with_user_and_mock_crate(
+        &app, ::new_user("bar"), "foo_not");
+    ::sign_in(&mut req, &app);
```

Comment:
Ah good point. We should probably have that endpoint return a 403 instead of a 200 with an error message, which would have caught this.